optimum/habana/peft/layer.py

```diff
@@ -2,6 +2,8 @@
 
 import torch
 
+import habana_frameworks.torch.hpex.experimental.transformer_engine as te
+from peft.utils.other import transpose
 
 def GaudiLoraLayerLinearForward(self, x: torch.Tensor, *args: Any, **kwargs: Any) -> torch.Tensor:
```

Comment:
If we end up creating a LoRALinear class, then we should unify this function with the new class.

Reply:
Latest peft doesn't require this special handling. so these changes are not required anymore since we moved to peft0.10.0